Source/Core/DiscIO/Blob.cpp

```diff
@@ -177,23 +178,24 @@ std::unique_ptr<IBlobReader> CreateBlobReader(const std::string& filename)
   if (cdio_is_cdrom(filename))
     return DriveReader::Create(filename);
 
-  if (!File::Exists(filename))
+  File::IOFile file(filename, "rb");
+  u32 magic;
```

Comment:
Gah, sorry @JosJuice  my comments were pending for like a week, didn't realize I didn't submit them.  Now I see @Helios747 had this same question.  So what is this magic number doing?